SystemDocumentation/CodeAccess.txt

```diff
@@ -0,0 +1,60 @@
+This document will give information about our groups code for perusal and printing.
+
+To access our groups code please unzip the POS.zip file and all of the files in here will be explained below.
+
+Below this will be a brief explanation of each file.
+
+For easy instructions on printing the files in the
+root directory you need to print would be:
+Database.py, DatabaseUrlGenerator.py, DatabaseUtils.py, DbConnection.py, UserLoginPackage.py, main.py, script.py
```

Comment:
May be worth adding `Errors.py` here as well. Even though it literally just defines error strings